### DISCUSSION TITLE
input range

### DISCUSSION THREAD

**Discussion by mauritskorse** [livewire/flux#242] [Feature Requests] [answered]
A range input slider with a single or with two handles (min max).
I found a pen here that sort of shows what I mean: https://codepen.io/t7team/pen/abNGeXB
some variants:
- having icons on one or either side, 
- the value at the end (or on either side in case of min-max slider)
- having tickmarks at certain intervals

**Comment by idaraty** [comment]
Great idea, and this can be a great starting point for inspiration: [advanced-range-slider](https://preline.co/docs/advanced-range-slider.html)

**Reply by Hesesses** [reply]
Would need this also 🙏

**Comment by dansan92** [comment]
Hoping this comes soon! 

I know it's in the plans 😉
https://x.com/calebporzio/status/1933203547924975666

**Comment by mauritskorse** [comment]
Nice, it's been added in v2.8.
Closing this request.

**Comment by joshhanley** [comment, marked as answer]
We've released the slider in v2.8.0!

See the blog post here https://fluxui.dev/blog/2025-11-25-slider
And you can find the documentation here https://fluxui.dev/components/slider
